electronics/electric_conductivity.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+ELECTRON_CHARGE = 1.6021e-19  # units = C
+5 * 2
+
+
+def electric_conductivity(
+    conductivity: float,
+    electron_conc: float,
+    mobility: float,
+) -> tuple:
+    """
+    This function can calculate any one of the three -
+    1. Conductivity
+    2, Conductivity
+    3. Electron Mobility
+    given the other two.
```

Comment:
This doesn't make sense and could be worded differently
```suggestion
    This is calculated from the other two provided values
```
Or perhaps this is just unnecessary information
```suggestion
```